From db4f84718b4e85286ce2f3b7d6b7e16781310f7c Mon Sep 17 00:00:00 2001
From: FrancescoFerraro <francesco.f@variscite.com>
Date: Sun, 2 Jan 2022 08:47:15 +0100
Subject: [PATCH 3/7] imx8mn: Remove uart2 and uart4 domain restrictions

i.MX8MN ATF code assigns UART2 to the A53 Domain and UART4 to the M7 domain.
Remove these assignments to allow for usage in either domain.

Cherry-picked from: b1bf9b5872d7

Signed-off-by: Nate Drude <nate.d@variscite.com>
Signed-off-by: Marco Contenti <marco.c@variscite.com>
---
 plat/imx/imx8m/imx8mn/imx8mn_bl31_setup.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/plat/imx/imx8m/imx8mn/imx8mn_bl31_setup.c b/plat/imx/imx8m/imx8mn/imx8mn_bl31_setup.c
index 22abd2448..9d1e562ad 100644
--- a/plat/imx/imx8m/imx8mn/imx8mn_bl31_setup.c
+++ b/plat/imx/imx8m/imx8mn/imx8mn_bl31_setup.c
@@ -108,8 +108,6 @@ static const struct imx_rdc_cfg rdc[] = {
 	RDC_MDAn(RDC_MDA_M7, DID1),
 
 	/* peripherals domain permission */
-	RDC_PDAPn(RDC_PDAP_UART4, D1R | D1W),
-	RDC_PDAPn(RDC_PDAP_UART2, D0R | D0W),
 	RDC_PDAPn(RDC_PDAP_RDC, D0R | D0W | D1R),
 
 	/* memory region */
-- 
2.34.1

